doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc

```diff
@@ -58,3 +58,32 @@ If the {CENTRAL} port is 8080, as in this example, then the {KIE_SERVER} port, w
 ====
 
 {KIE_SERVER} connects to the new {CENTRAL} and collects the list of deployment units (KIE containers) to be deployed.
+
+ifdef::PAM[]
+[NOTE]
+====
+When the class is used inside a dependency JAR file to access {KIE_SERVER} from {KIE_SERVER} client, you get the `ConversionException` and `ForbiddenClassException` in {CENTRAL}.
+To avoid generating these exceptions in {CENTRAL}, do any of the following tasks:
+
+* If the exceptions are generated at client side, add following system property to the kie-server client:
+[source]
+----
+System.setProperty("org.kie.server.xstream.enabled.packages", "org.example.**");
+----
+* If the exceptions are generated at server side, go to {PRODUCT} installation directory, open `standalone-full.xml` and under the <system-properties> tag, set the following property:
+[source]
+----
+<property name="org.kie.server.xstream.enabled.packages" value="org.example.**"/>
+----
+* Set the following JVM property:
+[source]
+----
+-Dorg.kie.server.xstream.enabled.packages=org.example.**
+----
+
+The `org.example` package is mandatory. You can specify multiple packages separated by comma , for example,
```

Comment:
What do you mean here? Should the user specify the package name as org.example?

Reply:
Yes, org.example is the main package name.

Reply:
I don't think it is mandatory, if a user doesn't have this package in his project, is it?

Reply:
org.example is not a mandatory package, it is just an example. User should define any package he wants.

Reply:
Ack.